tests/phpunit/tests/rest-api/rest-search-controller.php

```diff
@@ -368,9 +371,16 @@ public function test_get_items_search_prime_ids() {
 		$ids = wp_list_pluck( $response->get_data(), 'id' );
 		$this->assertSameSets( self::$my_primed_post_ids, $ids, 'Query result posts ids do not match with expected ones.' );
 
-		// Primed query will use WHERE ID IN clause.
-		$primed_query_string = 'WHERE ID IN (' . implode( ',', $ids ) . ')';
-		$this->assertStringContainsString( $primed_query_string, $wpdb->last_query, 'Last query do not use WHERE ID IN clause (posts not primed)' );
+		$args               = $action->get_args();
+		$primed_query_found = false;
+		foreach ( $args as $arg ) {
+			// Primed query will use WHERE ID IN clause.
+			if ( strpos( $arg[0], 'WHERE ID IN (' . implode( ',', $ids ) ) > 0 ) {
```

Comment:
Could we use `str_contains` here?

Reply:
str_contains is only available in [PHP 8.0](https://www.php.net/manual/en/function.str-contains.php)

Reply:
And in WP core since 5.9 - https://github.com/WordPress/wordpress-develop/commit/3cc8f1237a737ccf895682029afacafd7305d420

Reply:
I didn't know that, I will use it then.